README.md

```diff
@@ -4,12 +4,25 @@
 ![Codecov](https://img.shields.io/codecov/c/github/keptn/lifecycle-toolkit?token=KPGfrBb2sA)
 ![goversion](https://img.shields.io/github/go-mod/go-version/keptn/lifecycle-toolkit?filename=operator%2Fgo.mod)
 ![version](https://img.shields.io/github/v/release/keptn/lifecycle-toolkit)
-![status](https://img.shields.io/badge/status-not--for--production-red)
 [![GitHub Discussions](https://img.shields.io/github/discussions/keptn/lifecycle-toolkit)](https://github.com/keptn/lifecycle-toolkit/discussions)
 
 The goal of this toolkit is to introduce a more “cloud-native” approach for pre- and post-deployment, as well as the concept of application health checks.
 It is an incubating project, under the umbrella of the [Keptn Application Lifecycle working group](https://github.com/keptn/wg-app-lifecycle).
 
+Status of the different features:
+
+- ![status](https://img.shields.io/badge/status-stable-brightgreen) Observability
```

Comment:
I think it would make sense to describe the scope of the feature sets, e.g. "What can I assume to be working when observability is stable"